batch/batch/worker/worker.py

```diff
@@ -119,9 +121,16 @@
 docker: Optional[aiodocker.Docker] = None
 
 port_allocator: Optional['PortAllocator'] = None
+network_allocator: Optional['NetworkAllocator'] = None
 
 worker: Optional['Worker'] = None
 
+image_configs: Dict[str, Dict[str, Any]] = dict()
+
+
+def get_image_digest(image_config) -> str:
+    return image_config['RepoDigests'][0].split('@')[1].split(':')[1]
```

Comment:
Colons are kosher: 

```
(base) # touch :
(base) # mkdir ::
(base) # ls -al
total 0
drwxr-xr-x   4 dking  wheel  128 Jun 25 18:11 .
drwxrwxrwt  25 root   wheel  800 Jun 25 18:11 ..
-rw-r--r--   1 dking  wheel    0 Jun 25 18:11 :
drwxr-xr-x   2 dking  wheel   64 Jun 25 18:11 ::
```

We use ext4 and XFS, but basically every file system we'd ever consider (including networked ones like Lustre) support everything but `/`s. https://en.wikipedia.org/wiki/Comparison_of_file_systems

That said, I'm fine with splitting on the `:`, it gets the "encoded" part of the digest [according to the OCI format](https://github.com/opencontainers/image-spec/blob/master/descriptor.md#digests)